examples/fetch-embed-job-example/requirements.txt

```diff
@@ -0,0 +1,5 @@
+atlassian-python-api
+langchain_community
+lxml
+nltk
+sentence-transformers
```

Comment:
i think you should fix here

Reply:
I think the '-' or '_' cases for sentence transformers package are interchangeable.